ci_framework/roles/operator_build/molecule/default/molecule.yml

```diff
@@ -0,0 +1,13 @@
+---
+# Mainly used to override the defaults set in .config/molecule/
+# By default, it uses the "config_podman.yml" - in CI, it will use
+# "config_local.yml".
+log: true
+
+provisioner:
+  name: ansible
+  log: true
+  ansible_args:
+    - --skip-tags=build,push
```

Comment:
skipping tags while I don't figure it out how to properly test this.
We can replace with ci_make dry_run too.